### DISCUSSION TITLE
Can you delete ETLs in code?

### DISCUSSION THREAD

**Discussion by ItsRobbAllen** [ravendb/ravendb#19219] [Q&A] [answered]
We have a lot of ETLs that are no longer needed and I'd like to automate the removal of them, but I can't see anywhere how to delete an existing ETL. There is `AddEtlOperation` and `UpdateEtlOperation` but no Delete or Remove that I can find.

**Comment by arekpalinski** [comment, marked as answer]
There is `DeleteOngoingTaskOperation`. Sample usage in our tests:

https://github.com/ravendb/ravendb/blob/9dfea850b66c340a6c1f0ed13dfa350073967595/test/SlowTests/Server/Documents/ETL/EtlTasksManagement_RavenDB_7276.cs#L36-L47

**Comment by ItsRobbAllen** [comment]
Thank you so much. Missed that somehow.
